python/lsst/pipe/tasks/selectImages.py

```diff
@@ -309,11 +311,11 @@ def runDataRef(self, dataRef, coordList, makeDataRefList=True, selectDataList=[]
             scatterSize = sigmaMad(starSize - psfSize)
 
             valid = True
-            if medianE1 > self.config.maxEllipResidual:
+            if self.config.maxEllipResidual and medianE1 > self.config.maxEllipResidual:
                 self.log.info("Removing visit %s because median e residual too large: %f" %
```

Comment:
I'm fairly convinced @rearmstr meant `medianE` instead of `medianE1`, and so I've added a commit fixing that up (see the commit message for my reasoning).
I've also added a commit to include the limit when printing the value.